source/_docs/email.md

```diff
@@ -140,9 +140,14 @@ No, mail logs are not available for download and we do not recommend using the l
 ### What ports are recommended by Pantheon?
 Configuring mail to use port 25, 465 or 587 is strongly discouraged because they attract SPAM activities. Make sure that your service provider allows traffic on a port other than those mentioned and that you have correctly configured your site to use that port.
 
-### Can Pantheon provide, publish, or support SPF records?
+### Are there SPF records for Pantheon's local MTA (postfix)?
+If you are using Pantheon's local MTA ([not recommended](#outgoing-email)), and your domain contains an SPF record, then you should include Pantheon's SPF record, as shown below:
 
-As consumers of cloud infrastructure, we don’t have control over our IP ranges and they are subject to change without our notice. Publishing an SPF record would imply assurance on our end that it can work, which would be very difficult to guarantee given these circumstances. We take the decision of what we support and what we don’t very seriously, and at this time we’re not in a position to support SPF records.
+```
+v=spf1 include:spf.pantheon.io -all
+```
+
+Adjust the above example records as needed for your domain; be sure that you are configuring the SPF record for the domain that your site is set up to use in the `From` address of outgoing messages. If an SPF record exists for that domain, then add just the `include:spf.pantheon.io` part to whatever is already there, keeping the rest unchanged. To craft a new SPF record for a domain that does not yet have one, use the [SPF Record Generator](https://mxtoolbox.com/SPFRecordGenerator.aspx?domain=example.com), and enter `spf.pantheon.io` in the **3rd party mail systems** text box.
```

Comment:
>Adjust the above example records as needed for your domain;

@greg-1-anderson @ari-gold the only record displayed is the one we are suggesting they add for pantheon.io. How would that be adjusted based on the end-user's domain?

Reply:
@alexfornuto In addition to the text we show above, the user will need to add `mx` if they want to send email from the same servers that their mx records point at, or they might want to add the fqdn of their outgoing email servers if they are different than their receiving email servers, or they might want to add another `include` if they are sending through Google GSuite, etc.

Would it be clearer if we said "as needed for your use case" instead?  My feeling is that folks who really feel that they need spf records are already going to know what they need, and since this is not a recommended configuration, we shouldn't give too much of a tutorial to lead folks down a path that they don't need.

Reply:
It sounds like the clarification I'm looking for would come by replacing 'Adjust the above' with something like 'Include this record, along with any others needed for your configuration...'.

Thoughts?

Reply:
The key thing that we are trying to convey is that the user should add `include:spf.pantheon.io` to whatever else they need in their SPF record. I gave it as an example because this has to come after the `v=spf1` and before the `-all` (or `~all` or whatever form they wish to use).

Your suggestion sounds good.